### ISSUE TITLE
Config File ".powergate" Not Found

### ISSUE THREAD

**Issue by andrewxhill** [textileio/powergate#496]
```
▶ pow wallet new help  
> Success! Wallet address: t3vv5crhofq3wc2hx2spkwkou57nylfbqwdcegwr3amwhvkjyvufetdh5cyhprkispeibagt5fglh73dbg4wga
> Error! Config File ".powergate" Not Found in "[/Users/andrewhill]"
```

I'm getting this when building the binary and then manually moving to bin path. I built it when testing the new CI, so using the gox tool. I'm running devnet with the currently released docker image in Releases... which i see is 9 days old so perhaps is part of this.

**Comment by jsign** [comment]
I think we've been focused on `pow ffs` sub-world and missed that `pow init` was necessary to create the config file. (Such as other `init`s in our tools).

If you do:
```
pow init
pow wallet new
```
You won't have that error message. (Checked me)

cc @asutula  to double-check the rationale.
If that's the case, we can jump to add that `pow init` step to docs/readme.

**Comment by asutula** [comment]
Ooof, been a while. Didn't think the config file was required, but maybe for wallet related stuff it is. To keep track of which address to use, it may do some state management in the config file. Will look into it, and if it's required, it should just be created as needed automatically.

**Comment by jsign** [comment]
Addressed #499
